rca/landingpages/migrations/0028_add_related_events.py

```diff
@@ -0,0 +1,66 @@
+# Generated by Django 3.1.10 on 2021-07-26 07:40
```

Comment:
Is it possible to reduce these to one migration?

Reply:
Seen as all these are split tickets it's just easier to follow 1 migrations per PR for me